packages/@storefront/structure/test/unit/filtered-list.ts

```diff
@@ -70,30 +70,54 @@ suite('FilteredList', ({ expect, spy }) => {
   });
 
   describe('onKeyDown()', () => {
-    it('should select refinement', () => {
+    it('should exit early if the key is not Enter', () => {
+      const keyboardEvent = { keyCode: 1970 };
```

Comment:
Would it be better if we exported this value? I'm not sure if it's correct to do so, though.

Reply:
@jmbtutor you mean the target `keyCode`? I think it would make sense to define and expose a map of `keyCode` values, to be imported by any components which use them (ie. within `key*` handlers).

Reply:
There is now an open PR which updates how keyboard events are handled within StoreFront: https://github.com/groupby/storefront/pull/44. If that PR flows in first, I'll update this test case (and the function under test).